cns/restserver/internalapi.go

```
@@ -254,3 +262,57 @@ func (service *HTTPRestService) CreateOrUpdateNetworkContainerInternal(req cns.C
 	return returnCode
 
 }
+
+// Try to register node with DNC when CNS is started in managed DNC mode
+func RegisterNode(httpRestService cns.HTTPService, dncEP, infraVnet, nodeID string) {
```

This should return an error